homeassistant/components/iotty/manifest.json

```diff
@@ -7,5 +7,5 @@
   "documentation": "https://www.home-assistant.io/integrations/iotty",
   "integration_type": "device",
   "iot_class": "cloud_polling",
-  "requirements": ["iottycloud==0.2.1"]
+  "requirements": ["iottycloud==0.3.0"]
```

Comment:
bump this in a prerequisite PR

Reply:
Hi, it has been done here (pr already merged): https://github.com/home-assistant/core/pull/132836